tasks/go_test.py

```diff
@@ -283,7 +284,7 @@ def command(_empty_result, module, _module_result):
         # codecov gets confused and merges the roots of all modules, resulting in a
         # nonsensical directory tree in the codecov app
         path = os.path.normpath(os.path.join(module.path, PROFILE_COV))
-        ctx.run(f"codecov -f {path} -F {tag}", warn=True)
+        ctx.run(f"codecov -f {path} -F {tag} -t {token}", warn=True)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** It would be better to use an environnement variable for this‏

Reply:
:facepalm: my bad for this, thanks ! Fixed in [5d80746](https://github.com/DataDog/datadog-agent/pull/25363/commits/5d80746199cd93ee32e977a8b90d199451c28840)